src/polymesh/polymesh.service.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('PolymeshService', () => {
   afterAll(async () => {
     await service.close();
     await new Promise(resolve => setTimeout(resolve, 5000));
```

Comment:
IDK why we hold the test open for 5 seconds here.

Maybe better to try reducing the wait rather than increase the timeout